test/util/cnxml.js

```diff
@@ -1,11 +1,14 @@
-export default global.cnxml = function cnxml(fragments) {
-    if (fragments.length > 1) {
-        throw new Error('cnxml`` does not support interpolation')
-    }
-
+export default global.cnxml = function cnxml(content, glossary) {
     return `<document xmlns="http://cnx.rice.edu/cnxml">
```

Comment:
I don't like this change. It requires that we change all our serialization tests and write them in a more complex way, and the new addition is only ever used twice.

Reply:
This was the simplest and fastest solution which came to my mind. You are welcome to propose other, however I don't see this being more complex, especially when I already adjusted tests and probably there will be not many more new functions coming.

Reply:
The simplest solution would've been to just add a new function. As for it already being changed; this still makes tests harder to read and understand, and I don't agree that we won't change them in the future: Slate is still quire far from stable, and it's more than likely we'll have to adjust tests as it changes.

Reply:
I've rolled back those changes and adjusted tests, but I still consider this not intuitive since we allow two values in out main `CNXML` class.